test/var_compatibility_test/run_test

```diff
@@ -241,8 +247,8 @@ echo -e "\nChecking lists from command line"
 #check_datatable ${report_prog} ${datafile} "--process-list" ${process_list}
 check_datatable ${report_prog} ${datafile} "--module-list" ${module_list}
 #check_datatable ${report_prog} ${datafile} "--dependencies" ${dependencies}
-check_datatable ${report_prog} ${datafile} "--suite-list" ${suite_list}       \
-                --sep ";"
+#check_datatable ${report_prog} ${datafile} "--suite-list" ${suite_list}       \
+#                --sep ";"
```

Comment:
curious as to why this check was commented out?

Reply:
Because it didn't work? I'm not sure what changed but I fixed it and uncommented those checks.